fury/tests/test_ui.py

```diff
@@ -1197,6 +1197,89 @@ def _on_change():
         show_manager.start()
 
 
+def test_ui_file_dialog_2d(interactive=False):
+    filename = "test_ui_file_dialog_2d"
+    recording_filename = pjoin(DATA_DIR, filename + ".log.gz")
+    expected_events_counts_filename = pjoin(DATA_DIR, filename + ".json")
+
+    file_dialog_open = ui.FileDialog2D(os.getcwd(), draggable=False,
+                                       dialog_type="Open")
+
+    # Create temporary directory and files
+    os.mkdir(os.path.join(os.getcwd(), "testdir"))
```

Comment:
You could use something like `TemporaryDirectory` from `tempfile` to create temporary directory. That way it will automatically delete the temporary folder and files created under the temporary directory. Nothing critical.